frontend/src/citizen-frontend/messages/ThreadView.tsx

```diff
@@ -202,8 +215,10 @@ export default React.memo(function ThreadView({
               data-qa="message-reply-editor-btn"
               text={i18n.messages.replyToThread}
             />
+            <Gap size={'m'} />
           </>
         ))}
+      {replyEditorVisible && <span ref={messagesEndRef} />}
```

Comment:
Remove this.

Reply:
This was committed intentionally. I think it scrolls nicely to the bottom when the user wants to create a new message or a new message is sent.

Reply:
Ah ok. Sorry, I just misunderstood the intention.